dds/DCPS/Service_Participant.cpp

```diff
@@ -428,15 +428,10 @@ Service_Participant::get_domain_participant_factory(int &argc,
 #endif
 
       if (DCPS_debug_level > 0) {
-        String version_string = OPENDDS_VERSION;
-        const char* const version_metadata = "-" OPENDDS_VERSION_METADATA;
-        if (version_metadata[1] != '\0') {
-          version_string += version_metadata;
-        }
         ACE_DEBUG((LM_NOTICE,
                    ACE_TEXT("(%P|%t) NOTICE: Service_Participant::get_domain_participant_factory - ")
                    ACE_TEXT("This is OpenDDS %C using ACE %C.\n"),
-                   version_string.c_str(), ACE_VERSION));
+                   OPENDDS_VERSION, ACE_VERSION));
```

Comment:
Do we want to normalize these to the approach that `be_init.cpp` uses where strings that are known at compile time (and known not have special characters) are directly part of the format string and not a separate parameter?